tools/amp_segment/keep_speech.xml

```diff
@@ -24,7 +24,7 @@
   <help>
 .. class:: infomark
 
-Remove segments of silence and music from an audio file.
+Remove segments of silence, noise and music from an audio file.
 
   </help>
 </tool>
```

Comment:
Let's keep the original remove_silence_music MGM intact (including its python scipts), as they are being used in staging in some unfinished workflows. We can add new tool keep_speech along with its own python code. The old tools will be removed from tool panel, and we can inform create new workflows with new tool.

Reply:
Good thought.  I added the old ones back in, but kept the reference in tool_conf.xml to the new one.